### PR TITLE
add weight input

### PR DESCRIPTION
**Pull request by tiltec**
Refs #159 request from @nicco11a
input box is still a bit misplaced...

image

I tried to add `class="row"` but the damn thing didn't move...

### CODE REVIEW

**Comment by D0nPiano** [comment]
Makes sense to have an input field there (just forgot to put one there :smile: ) Just improved some style & handling of very large numbers

scale

File: src/components/Statistics/AmountPicker.vue [hunk 1]
```diff
@@ -16,28 +16,55 @@
...
+      set (v) {
+        this.value = v
+      },
+    },
+    valueToNumber: {
+      get () {
+        return this.value
+      },
+      set (v) {
+        const value = parseFloat(v, 10)
```
Comment (D0nPiano) [inline src/components/Statistics/AmountPicker.vue]: Just changed parseInt to parseFloat - or was there a specific reason to use Int?

Comment (tiltec) [reply src/components/Statistics/AmountPicker.vue]: by mistake :)

File: src/components/Statistics/AmountViewer.vue [hunk 2]
```diff
@@ -67,6 +70,9 @@ export default {
         }
         amount -= 0.15
       }
+      if (amountImages.length > 6) {
```
Comment (tiltec) [inline src/components/Statistics/AmountViewer.vue]: shouldn't this be `> 9`?

Comment (tiltec) [reply src/components/Statistics/AmountViewer.vue]: I think you can leave the check out, slice does it for you:

> If end is greater than the length of the sequence, slice extracts through the end of the sequence (arr.length).

**Pull request merged**